src/sections/evidence/KidsFirst/Body.js

```diff
@@ -0,0 +1,140 @@
+import React, { useEffect, useState } from 'react';
+import { Tab, Tabs } from '@material-ui/core';
+import { getGeneDiseaseGtexPlot } from '../../../utils/externalAPI';
+import SectionItem from '../../../components/Section/SectionItem';
+import Description from './Description';
+import DataDownloader from '../../../components/Table/DataDownloader';
+import { Grid } from '@material-ui/core';
+import { dataTypesMap } from '../../../dataTypes';
+
+import {getData} from './Summary'; 
+
+function Body({ definition, id, label }) {
+  const { ensgId: ensemblId, efoId } = id;
+
+  const [json, setJson] = useState([])
+  const [linearPlot, setLinearPlot] = useState('')
+  const [log10Plot, setLog10Plot] = useState('')
+  const [tab, setTab] = useState('linear');
+  const [loading, setLoading] = useState(true)
+  const [error, setError] = useState(false)
+  const [hasData, setHasData] = useState(false)
+
+  const handleOnChange = (_, tab) => {
+    return setTab(tab);
+  };
+
+  useEffect(()=>{
+    // Get JSON Data and use this data to determine if TPM plot exist or not
+    getData(id, setJson, setLoading, setHasData)
+  }, [ensemblId, efoId, id])
+  useEffect(
+    ()=>{
+      if (hasData && tab === 'linear' && linearPlot.length === 0) {
+        /********     Get Plot    ******** */ 
+        setLoading(true)
+        getGeneDiseaseGtexPlot(ensemblId, efoId, tab, 
+          (resData) => {
+            const base64 = Buffer.from(resData).toString('base64')
+            const imageSrc = base64
+            setLinearPlot(imageSrc)
+            setLoading(false)
+          },
+          (error)=> {
+            setLoading(false)
+            setError(true)
+            console.log(error)
+          });
+      } else if (hasData && tab === 'log10' && log10Plot.length === 0) {
+          /********     Get Plot    ******** */ 
+          setLoading(true)
+          getGeneDiseaseGtexPlot(ensemblId, efoId, tab,  
+            (resData) => {
+              const base64 = Buffer.from(resData).toString('base64')
+              const imageSrc = base64
+              setLog10Plot(imageSrc)
+              setLoading(false)
+            },
+            (error)=> {
+              setLoading(false)
+              setError(true)
+              console.log(error)
+            });
+      }
+    }, [efoId, hasData, ensemblId, tab]);
```

Comment:
Codacy has a fix for the issue: [React Hook useEffect has missing dependencies: 'linearPlot.length' and 'log10Plot.length'. Either include them or remove the dependency array.](https://app.codacy.com/gh/CBIIT/ppdc-otp-frontend/pullRequest?prid=8203564)
```suggestion
    }, [efoId, hasData, ensemblId, tab, linearPlot.length, log10Plot.length]);
```